README.md

```diff
@@ -1,4 +1,6 @@
 # FastChat
+![GitHub pull requests](https://img.shields.io/github/issues-pr/lm-sys/FastChat?style=plastic) ![GitHub closed pull requests](https://img.shields.io/github/issues-pr-closed-raw/lm-sys/FastChat) ![GitHub issue](https://img.shields.io/github/issues/lm-sys/FastChat?style=plastic) ![GitHub closed pull issue](https://img.shields.io/github/issues-closed-raw/lm-sys/FastChat)
```

Comment:
This isn't particularly helpful. We might not want it.

Reply:
Thank you. Do we have any issues that I can work on, or any direction that I can contribute to these repositories? I am mostly refactoring small things most of the time.

Reply:
Let us close this PR for now. We will create some good first issues for you later if you want to contribute to this repo.

Reply:
@HiteshGorana Could you help with this? https://github.com/lm-sys/FastChat/issues/254

Reply:
sure!! Thank you